etcdserver/api/v3rpc/watch.go

```diff
@@ -61,11 +62,16 @@ func (ws *watchServer) Watch(stream pb.Watch_WatchServer) error {
 func sendLoop(stream pb.Watch_WatchServer, watcher storage.Watcher, closec chan struct{}) {
 	for {
 		select {
-		case e, ok := <-watcher.Chan():
+		case evs, ok := <-watcher.Chan():
+			// evs is []storagepb.Event type
 			if !ok {
 				return
 			}
-			err := stream.Send(&pb.WatchResponse{Event: &e})
+			events := make([]*storagepb.Event, len(evs))
```

Comment:
this does not make a lot of sense. We probably need to change the chan type to pointer type. Add a todo here?